crmsh/utils.py

```diff
@@ -2779,6 +2779,15 @@ def get_property(name, property_type="crm_config", peer=None):
     return stdout if rc == 0 else None
 
 
+def delete_property(name, property_type="crm_config"):
```

Comment:
Should provide a feedback about whether the operation succeeds. Please add a return value or raise an exception when it fails.